java/java.graph/src/org/netbeans/modules/java/graph/NodeWidget.java

```diff
@@ -78,13 +77,18 @@
     private static final int RIGHT_TOP = 3;
     private static final int RIGHT_BOTTOM = 4;
 
-    private static final @StaticResource String LOCK_ICON = "org/netbeans/modules/java/graph/resources/lock.png";
-    private static final @StaticResource String LOCK_BROKEN_ICON = "org/netbeans/modules/java/graph/resources/lock-broken.png";
-    private static final @StaticResource String BULB_ICON = "org/netbeans/modules/java/graph/resources/bulb.gif";
-    private static final @StaticResource String BULB_HIGHLIGHT_ICON = "org/netbeans/modules/java/graph/resources/bulb-highlight.gif";
+    private static final @StaticResource
+    String LOCK_ICON = "org/netbeans/modules/java/graph/resources/lock.png";
+    private static final @StaticResource
+    String LOCK_BROKEN_ICON = "org/netbeans/modules/java/graph/resources/lock-broken.png";
+    private static final @StaticResource
+    String BULB_ICON = "org/netbeans/modules/java/graph/resources/bulb.gif";
+    private static final @StaticResource
+    String BULB_HIGHLIGHT_ICON = "org/netbeans/modules/java/graph/resources/bulb-highlight.gif";
```

Comment:
These formatting changes can be left out of the patch.

Reply:
My Netbeans autoformatter has different setting than yours!